docs/asyncapi.yaml

```diff
@@ -115,6 +119,212 @@ components:
                 accountCode:
                   type: string
                   description: Unique account code to pay from
+    ManualLedgerCheck:
+      contentType: application/json
+      payload:
+        type: object
+        properties:
+          deltaPaymentRequest:
```

Comment:
Can we strip this right back to only having properties which can/will be changed by automated ledger assignment?